### PR TITLE
fix lint errors & bumps eslint-plugin-react to 7.20.6

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves n/a

**Overall change:**
_fixes lint errors and bumps eslint-plugin-react to 7.20.6._

**Code changes:**
_fixes linter errors and bumps eslint-plugin-react to 7.20.6._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by karinathomasbbc** [comment]
Do you also need to bump the version of eslint-plugin-react from 7.20.5 to 7.20.6?

**Comment by rhenshaw56** [comment]
> Do you also need to bump the version of eslint-plugin-react from 7.20.5 to 7.20.6?

yeah, will do that to see if the errors persist.

File: package-lock.json [hunk 1]
```diff
@@ -22937,6 +22937,15 @@
...
     },
+    "prismjs": {
+      "version": "1.17.1",
+      "resolved": "https://registry.npmjs.org/prismjs/-/prismjs-1.17.1.tgz",
+      "integrity": "sha512-PrEDJAFdUGbOP6xK/UsfkC5ghJsPJviKgnQOoxaDbBjwc8op68Quupwt1DeAFoG8GImPhiKXAvvsH7wDSLsu1Q==",
+      "dev": true,
+      "requires": {
+        "clipboard": "^2.0.0"
+      }
+    },
```
Comment (karinathomasbbc) [inline package-lock.json]: This shouldn't be here 😱

**Review by jroebu14 — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (rhenshaw56) [reply package-lock.json]: removed.

**Pull request merged**